.github/labeler.yml

```diff
@@ -0,0 +1,164 @@
+# to add labels to pull requests. This is not (yet?) a replacement for human
+# triaging, but is intended to add labels to the easy cases. If the matching
+# language becomes more powerful, more cases should be able to be handled.
+#
+# The biggest low-hanging problem is this:
+# It looks like there's no way of specifying that a label be added if *all* the
+# files match *any* one of a number of globs. This feature request is tracked
+# in https://github.com/actions/labeler/issues/423
+
+authentication:
+- all: ['lib/*gssapi*']
+- all: ['lib/*krb5*']
+- all: ['lib/*ntlm*']
+- all: ['lib/curl_sasl.*']
+- all: ['lib/http_aws*']
+- all: ['lib/http_digest.*']
+- all: ['lib/http_negotiate.*']
+- all: ['lib/vauth/**']
+
+build:
+- all: ['**/CMakeLists.txt']
+- all: ['**/Makefile.am']
+- all: ['**/*.inc']
+- all: ['**/*.m4']
+- all: ['**/*.mk']
+- all: ['configure.ac']
+- all: ['m4/**']
+
+CI:
+- all: ['appveyor.yml']
+- all: ['.azure-pipelines.yml']
+- all: ['.circleci/**']
+- all: ['.cirrus.yml']
+- all: ['.github/**']
+- all: ['zuul.d/**']
+
+cmake:
+- all: ['**/CMakeLists.txt']
+
+cmdline tool:
+- all: ['src/**']
+
+connecting & proxies:
+- all: ['lib/http_proxy.*']
+- all: ['lib/noproxy.*']
+- all: ['lib/socks.*']
+
+cookies:
+- all: ['lib/cookie.*']
+- all: ['lib/psl.*']
+
+cryptography:
+- all: ['lib/*sha256*']
+- all: ['lib/curl_hmac.*']
+- all: ['lib/curl_md?.*']
+- all: ['lib/md?.*']
+
+DICT:
+- all: ['lib/dict.*']
+
+documentation:
+- all: ['**/*.md']
+- all: ['**/*.txt', '!**/CMakeLists.txt']
+- all: ['**/*.1']
+- all: ['**/*.3']
+- all: ['docs/**', '!docs/examples/**']
+- all: ['RELEASE-NOTES']
+
+FTP:
+- all: ['lib/curl_range.*']
+- all: ['lib/ftp*']
+
+GOPHER:
+- all: ['lib/gopher*']
+
+HTTP:
+- all: ['lib/h2h3.*']
+- all: ['lib/http*']
+
+HTTP/3:
+- all: ['lib/vquic/**']
+
+HTTP/2:
+- all: ['lib/http2*']
+
+Hyper:
+- all: ['lib/c-hyper.*']
+
+IMAP:
+- all: ['lib/imap*']
+
+LDAP:
+- all: ['lib/*ldap*']
+
+libcurl API:
+- all: ['include/curl/**']
+
+MIME:
+- all: ['lib/mime*']
+
+MQTT:
+- all: ['lib/mqtt*']
+
+name lookup:
+- all: ['lib/asyn*']
+- all: ['lib/curl_gethostname.*']
+- all: ['lib/doh*']
+- all: ['lib/host*']
+- all: ['lib/idn*']
+- all: ['lib/socketpair*']
+
+POP3:
+- all: ['lib/pop3.*']
+
+RTMP:
+- all: ['lib/curl_rtmp.*']
+
+RTSP:
+- all: ['lib/rtsp.*']
+
+SCP/SFTP:
+- all: ['lib/vssh/**']
+
+script:
+- all: ['scripts/**']
+- all: ['**/*.pl']
+- all: ['**/*.sh']
+
+SMB:
+- all: ['lib/smb.*']
+
+SMTP:
+- all: ['lib/smtp.*']
+
+tests:
+- all: ['tests/**']
+
+TFTP:
+- all: ['lib/tftp.*']
+
+TLS:
+- all: ['lib/vtls/**']
+
+URL:
+- all: ['lib/urlapi*']
+- all: ['lib/url.*']
+
+WebSocket:
+- all: ['lib/ws.*']
+- all: ['docs/WebSockets.md']
+- all: ['docs/libcurl/opts/CURLOPT_WS_OPTIONS.3']
+- all: ['docs/libcurl/curl_ws_*']
+
+Windows:
+- all: ['projects/Windows/**']
+- all: ['lib/*win32*']
+- all: ['lib/curl_multibyte.*']
+- all: ['lib/rename.*']
+- all: ['Winbuild/**']
```

Comment:
If this is case sensitive then it is 'winbuild' in lowercase. Also I think it should be added to the build tag if the labeler allows multiple tags to be applied to the same path.

Reply:
Done and done.